src/env/development.js

```diff
@@ -62,6 +62,7 @@ import { Pairwise } from '../examples/pairwise';
 import { Repeater } from '../examples/repeater';
 import { Table } from '../examples/table';
 import { TableCsv } from '../examples/table_csv';
+import { Ranker } from '../examples/ranker';
```

Comment:
Ah, I see, this example was not added to git. `src/examples` are copied from `examples` for more flexibility in modifying them. But at the end the final working version of example should be added to root `examples` dir.

Reply:
oh i see! I added it to the wrong examples directory, will fix that in my next commit.